js/cardview.js

```diff
@@ -1,11 +1,13 @@
 // React view of a card
 import React from "react";
-import GameStyle from "./style";
+require("../scss/style.scss");
```

Comment:
Hmmm... if it didn't work, I don't think the cards could be styled? What makes you suspect it doesn't work?

Maybe relatedly... because of how webpack chains the loaders, if you set up your webconfig as told to be sass-loader, the jsx code for including is shorter than if you don't.